foundation/foundation-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -66,11 +66,29 @@
 				<include>jenkinsPipelineSteps.groovy</include>
 				<include>.mvn/**</include>
 			</includes>
+			<excludes>
+				<exclude>mvnw</exclude>
+				<exclude>mvnw.cmd</exclude>
+			</excludes>
 		</fileSet>
 		<fileSet>
 			<directory>.mvn</directory>
 			<includes/>
 		</fileSet>
+		<fileSet filtered="false" encoding="UTF-8">
```

Comment:
I (Informational): filtering breaks the variables in the mvnw scripts, so turn it off when copying those files